app/build.gradle

```diff
@@ -53,10 +53,7 @@ android {
         versionCode versions.vCode
         versionName versions.vName
         testInstrumentationRunner "org.dhis2.Dhis2Runner"
-        vectorDrawables.useSupportLibrary = true
-        multiDexEnabled true
 
-        setProperty("archivesBaseName", "dhis2-v" + versions.vName)
```

Comment:
I think that you shouldn't remove these three lines from the file, they are needed